cantabular/contract.go

```diff
@@ -158,6 +158,18 @@ type GetParentsResponse struct {
 	Dataset gql.Dataset `json:"dataset"`
 }
 
+// GetCategorisationsRequest holds the input parameters for the GetCategorisations query
+type GetCategorisationsRequest struct {
+	PaginationParams
+	Dataset  string
+	Variable string
+}
+
+// GetCategorisationsResponse is the response body for the GetCategorisations query
+type GetCategorisationsResponse struct {
+	Dataset gql.Dataset `json:"dataset"`
+}
+
```

Comment:
Do we need to know the `totalCount` and this `count` batch?

Reply:
Yes you're right. Now added in response